tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -10,3 +10,10 @@ tests:
       fixture: spi_loopback
   driver.spi.loopback.internal:
     filter: CONFIG_SPI_LOOPBACK_MODE_LOOP
+  drivers.spi_dma.loopback:
+    tags: dma
+    harness: ztest
+    extra_args: OVERLAY_CONFIG="overlay-mcux-dspi-dma.conf"
```

Comment:
Since this is effectively a test for 1 platform, which is implied by the overlay, it might make sense to add a platform component to the name of the test.

Reply:
right, let me correct it.